spec/controllers/miq_report_controller/trees_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'spec_helper'
+
+describe ReportController do
+  render_views
+  before :each do
+    set_user_privileges
+    FactoryGirl.create(:vmdb_database)
+    EvmSpecHelper.create_guid_miq_server_zone
+  end
+
+  context 'Reports #tree_select' do
+    it 'renders list of Reports in Reports - Custom tree' do
```

Comment:
@bmclaughlin you can add similar tests for other accordions to verify correct partial is rendered when clicking on tree nodes.